setup.py

```diff
@@ -98,6 +98,7 @@
         'scripts/dropstat',
         'scripts/dualtor_neighbor_check.py',
         'scripts/dump_nat_entries.py',
+        'scripts/debug_voq_chassis_packet_drops.sh',
```

Comment:
I thought this was flagged not in alphanumeric order and will be an issue?  Can you check with original change and perhaps with Arvind on this one?

Reply:
@gechiang, we will fix it in master in another PR. We can merge this PR for now.

Reply:
Follow this plan